### DISCUSSION TITLE
Have Original Year Show in Details Pane

### DISCUSSION THREAD

**Discussion by Groxxxy** [TT-ReBORN/Georgia-ReBORN#79] [Ideas]
Hi! I propose an additional default entry to the metadata grid.

`{ "label": "Original Year", "val": "$puts(d,$if3(%original release date%,%originaldate%,%fy_upload_date%,))$if($strcmp($year($get(d)),$get(d)),$get(d),)" }, // 'Original Year' is shown if %originaldate% exists.
		{ "label": "Year", "val": "$puts(d,$if3(%date%,%fy_upload_date%,))$if($strcmp($year($get(d)),$get(d)),$get(d),)" }, // 'Year' is shown if the date format is YYYY`

![foobar2000_BX8sC5wnac](https://user-images.githubusercontent.com/14025275/194767355-20214564-fca8-4352-8280-5d5b61be3950.jpg)

**Comment by TT-ReBORN** [comment]
Hi @Groxxxy, 

actually Mordred has already made it if you use %original release date%, you will have a nice edition tag with the date tag 
and year will be original release date. I'll also add the additional %originaldate% tag and then we have it complete.

It looks like this:

![edition](https://user-images.githubusercontent.com/79368795/194789111-d0e94ab4-271f-44fd-8053-3c438e39be17.jpg)


-Tom

**Reply by TT-ReBORN** [reply]
You can try it yourself, open the config and go to section and replace both:

"title_format_strings": {
`"edition": "[$if2($if(%original release date%,$ifequal($year(%original release date%),$year(%date%),,$year(%date%) ))$if2(%edition%,'release'),$if(%originaldate%,$ifequal($year(%originaldate%),$year(%date%),,$year(%date%) ))$if2(%edition%,'release'))]",`

"metadataGrid": [
`{ "label": "Edition", "val": "[$if2($if(%original release date%,$ifequal($year(%original release date%),$year(%date%),,$year(%date%) ))$if2(%edition%,'release'),$if(%originaldate%,$ifequal($year(%originaldate%),$year(%date%),,$year(%date%) ))$if2(%edition%,'release'))]" },`


This should be the correct pattern, the previous posted was not completely right...

-Tom

**Comment by Groxxxy** [comment]
Thank you Tom. Will the %original date% version be added to the next beta?

**Reply by TT-ReBORN** [reply]
Yes, already implemented it!
